.circleci/config.yml

```diff
@@ -54,6 +55,10 @@ jobs:
 
       # run go tests with gotestsum
       - run: |
+          # download and install the consul binary
+          wget https://releases.hashicorp.com/consul/"${CONSUL_VERSION}"/consul_"${CONSUL_VERSION}"_linux_amd64.zip && \
+               unzip consul_"${CONSUL_VERSION}"_linux_amd64.zip -d /home/circleci/bin &&
+               rm consul_"${CONSUL_VERSION}"_linux_amd64.zip
```

Comment:
This is a bit hacky, but I didn't want to break the CI.

Reply:
❤️ it!